src/css/custom.scss

```diff
@@ -0,0 +1,56 @@
+:root {
+  --ifm-color-primary: #1e4768;
```

Comment:
Used their colour generator with our blue and adjusted until it passed AAA on each colour.